MPF.Processors/Redumper.cs

```diff
@@ -75,20 +75,18 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, string basePath
                     if (!string.IsNullOrEmpty(cdMultiSessionInfo))
                         info.CommonDiscInfo.CommentsSpecialFields![SiteCode.Multisession] = cdMultiSessionInfo;
 
-                    // Attempt to get the universal hash, if it's an audio disc
-                    if (System.IsAudio())
+                    // Attempt to get extra metadata if it's an audio disc
+                    if (IsAudio(info.TracksAndWriteOffsets.Cuesheet))
                     {
                         string universalHash = GetUniversalHash($"{basePath}.log") ?? string.Empty;
-                        info.CommonDiscInfo.CommentsSpecialFields![SiteCode.UniversalHash] = universalHash;
-                    }
-
-                    // Attempt to get the non-zero data start, if it's an audio disc
-                    if (System.IsAudio())
-                    {
+                        if (!string.IsNullOrEmpty(universalHash))
```

Comment:
If you're adding this check, you can change the line above to be `string?` and remove the `?? string.Empty;`. Same with all of the below checks.